Pages/loginPage.py

```diff
@@ -26,8 +25,6 @@ def click_submit(self):
 
     def accept_alert(self):
         try:
-            WebDriverWait(self.driver, 10).until(
-                ec.presence_of_element_located((By.ID, self.alert_button_accept))).click()
-            time.sleep(7)
-        except Exception as e:
-            print(e)
+            self.driver.find_element_by_id(self.alert_button_accept).click()
+        except NoSuchElementException:
+            pass  # ignore if alert not on page
```

Comment:
@kiran-tripathy let me know if this does not work on your system. We may be able to take a different approach if it does not.

Reply:
It should be called there, I added it:

https://github.com/dimagi/dimagi-qa/blob/e330af3afb62bcaa49f6061d357ac06d03cfd778/TestBase/environmentSetupPage.py#L25

Reply:
My bad, I hadn't pulled your changes. This was still failing test cases, si I added 
https://github.com/dimagi/dimagi-qa/commit/978e2d7f1ad3564378cfecd69f71c2f72bccfbd5